docs/sources/alerting/unified-alerting/message-templating/_index.md

```diff
@@ -84,14 +113,20 @@ Template to render a single alert:
 Template to render entire notification message:
 
 ```
-{{ define "message" }}
+{{ define "mymessage" }}
   {{ if gt (len .Alerts.Firing) 0 }}
     {{ len .Alerts.Firing }} firing:
-    {{ range .Alerts.Firing }} {{ template "alert" .}} {{ end }}
+    {{ range .Alerts.Firing }} {{ template "myalert" .}} {{ end }}
   {{ end }}
   {{ if gt (len .Alerts.Resolved) 0 }}
     {{ len .Alerts.Resolved }} resolved:
-    {{ range .Alerts.Resolved }} {{ template "alert" .}} {{ end }}
+    {{ range .Alerts.Resolved }} {{ template "myalert" .}} {{ end }}
   {{ end }}
 {{ end }}
 ```
+
+### HTML in Message Templates
+
+HTML in Alerting message templates is escaped, and rendering of HTML in the resulting notification is not supported.
+
+Some notifiers which support rendered HTML (such as the body of emails) support alternative methods of changing the look and feel of the resulting notification. For example, the base template used in Alerting emails is installed to `/usr/share/grafana/public/emails` on Linux.
```

Comment:
```suggestion
Some notifiers that support rendered HTML (in the body of emails) support alternative methods of changing the look and feel of the resulting notification. For example, it may install the base template for alerting emails to `/usr/share/grafana/public/emails` on Linux.
```

Reply:
On second read, my wording in this whole block isn't the greatest. I only mentioned one OS, totally forgot the filename etc. How about:

Some notifiers support alternative methods of changing the look and feel of the resulting notification. For example, Grafana installs the base template for alerting emails to `<grafana-install-dir>/public/emails/ng_alert_notification.html`.